msi/release.sh

```diff
@@ -170,15 +170,16 @@ echo "$LIST" |
 sort |
 uniq |
 tr / \\\\ |
-sed -e 's/\(.*\)\\\(.*\)/            <Component Directory="INSTALLFOLDER:\\\1\\">\
+sed -r -e 's/(.*)\\(.*)/            <Component Directory="INSTALLFOLDER:\\\1\\">\
                 <File Source="&" \/>\
             <\/Component>/' \
-	-e 's/^\([^\\]*\)$/            <Component Directory="INSTALLFOLDER">\
-            <File Source="&" \/>\
-        <\/Component>/' \
-	-e 's/\(<File Source="git-bash.exe"[^>]*\) \/>/\1 \/><Shortcut Name="Git Bash" Icon="git.ico" Directory="GitProgramMenuFolder" WorkingDirectory="INSTALLFOLDER" Arguments="--cd-to-home" Advertise="yes" \/>/' \
-	-e 's/\(<File Source="git-cmd.exe"[^>]*\) \/>/\1 \/><Shortcut Name="Git CMD" Icon="git.ico" Directory="GitProgramMenuFolder" WorkingDirectory="INSTALLFOLDER" Arguments="--cd-to-home" Advertise="yes" \/>/' \
-	-e 's/\(<File Source="cmd\\git-gui.exe"[^>]*\) \/>/\1 \/><Shortcut Name="Git GUI" Icon="git.ico" Directory="GitProgramMenuFolder" WorkingDirectory="INSTALLFOLDER" Arguments="--cd-to-home" Advertise="yes" \/>/'
+	-e 's/^([^\\]*)$/            <Component Directory="INSTALLFOLDER">\
+                <File Source="&" \/>\
+            <\/Component>/' \
+	-e "s/<File Source=\"(mingw$BITNESS.*(ca)-(bundle\.crt))\" \/>/<File Id=\"\2_\3\" Source=\"\1\" \/>/"\
+	-e 's/(<File Source="git-bash.exe"[^>]*) \/>/\1 \/><Shortcut Name="Git Bash" Icon="git.ico" Directory="GitProgramMenuFolder" WorkingDirectory="INSTALLFOLDER" Arguments="--cd-to-home" Advertise="yes" \/>/' \
+	-e 's/(<File Source="git-cmd.exe"[^>]*) \/>/\1 \/><Shortcut Name="Git CMD" Icon="git.ico" Directory="GitProgramMenuFolder" WorkingDirectory="INSTALLFOLDER" Arguments="--cd-to-home" Advertise="yes" \/>/' \
+	-e 's/(<File Source="cmd\\git-gui.exe"[^>]*) \/>/\1 \/><Shortcut Name="Git GUI" Icon="git.ico" Directory="GitProgramMenuFolder" WorkingDirectory="INSTALLFOLDER" Arguments="--cd-to-home" Advertise="yes" \/>/'
```

Comment:
Oh, I get it. You use extended regexps. But they are not really needed, are they? This hunk should reach much easier if you skipped the `-r` option and simply added backslashes in front of the parentheses in the ca-bundle pattern.